MSAL/src/native_auth/network/responses/validator/sign_up/MSALNativeAuthSignUpResponseValidator.swift

```diff
@@ -171,14 +176,14 @@ final class MSALNativeAuthSignUpResponseValidator: MSALNativeAuthSignUpResponseV
     ) -> MSALNativeAuthSignUpContinueValidatedResponse {
         switch result {
         case .success(let response):
-            // Even if the `signInSLT` is nil, the signUp flow is considered successfully completed
-            return .success(response.signinSLT)
+            // Even if the `continuationToken` is nil, the signUp flow is considered successfully completed
+            return .success(response.continuationToken)
         case .failure(let error):
             return handleContinueError(error, with: context)
         }
     }
 
-    // swiftlint:disable:next cyclomatic_complexity
+    // swiftlint:disable:next cyclomatic_complexity function_body_length
```

Comment:
We should add this to the engineering backlog so we track that we want to refactor it

Reply:
Please, feel free to create a new PBI ;)